iTunes Search/Controllers/SearchTableVC.swift

```diff
@@ -0,0 +1,92 @@
+//
+//  SearchTableVC.swift
+//  iTunes Search
+//
+//  Created by Jeffrey Santana on 8/6/19.
+//  Copyright © 2019 Lambda. All rights reserved.
+//
+
+import UIKit
+
+class SearchTableVC: UITableViewController {
+	
+	//MARK: - IBOutlets
+	
+	@IBOutlet weak var categorySegControl: UISegmentedControl!
+	@IBOutlet weak var searchBar: UISearchBar!
+	
+	//MARK: - Properties
+	
+	private var searchResultsController = SearchResultController()
+	
+	//MARK: - Life Cycle
+	
+	override func viewDidLoad() {
+		super.viewDidLoad()
+		
+		searchBar.delegate = self
+	}
+	
+	//MARK: - IBActions
+	
+	@IBAction func categoryChange(_ sender: Any) {
+		searchMode()
+	}
+	
+	//MARK: - Helpers
+	
+	private func searchMode() {
+		guard let searchText = searchBar.optionalText else { return }
+		var resultType: ResultType {
+			switch categorySegControl.selectedSegmentIndex {
+			case 0:
+				return ResultType.app
+			case 1:
+				return ResultType.music
+			case 2:
+				return ResultType.movie
+			default:
+				return ResultType.all
+			}
+		}
+		
+		searchResultsController.performSearch(forSearch: searchText, ofResult: resultType) { (error) in
+			if let _ = error {
+				
```

Comment:
I would suggest at least putting a comment here reminding you to address the error in a future revision. I agree that it doesn't matter too much for a school assignment, but make sure you at least acknowledge that it would be something you'd do here.